{
  try {
    XQQueryException ex=new XQQueryException("Hello world!",new QName("foo"));
    assertEquals("A-XQQEX-1.1: XQQueryException successfully created","Hello world!",ex.getMessage());
    assertEquals("A-XQQEX-1.1: XQQueryException successfully created",new QName("foo"),ex.getErrorCode());
  }
 catch (  Exception e) {
    fail("A-XQQEX-1.1: Creating XQQueryException faild with message: " + e.getMessage());
  }
}
